### ISSUE TITLE
Coloured beds from custom objects not coloured in Minecraft

### ISSUE THREAD

**Issue by Captain-Chaos** [Captain-Chaos/WorldPainter#71]
Coloured beds from custom objects are all red in Minecraft, even though WorldPainter does export the tile entity metadata.

**Comment by worldwideguy** [comment]
has this still not been resolved? or are we looking for 1.13 to hopefully fix this?

**Comment by Captain-Chaos** [comment]
No, it's a difficult problem to track down. I'm working full-time again and will have far less time to spend on WorldPainter. 1.13 will definitely solve the problem, but 1.13 support in WorldPainter may take a little while yet, as the file format has changed drastically.

**Comment by worldwideguy** [comment]
thanks for getting back to me and i figured 1.13 would fix a lot of these block data issues. looking forward for 1.13 WP support so I can start building the map for my project! don't sweat it and don't overwork yourself! 

We appreciate all that you do
